juju-sync-watch

```diff
@@ -39,8 +42,9 @@ def parse_args():
     parser.add_argument('unit')
     parser.add_argument('charm_path', nargs='?')
     parser.add_argument('-d', '--description', action=ShowDesc, help='show description')
-    parser.add_argument('-e', '--environment', dest='env', default=JUJU_ENV,
-                        help='juju environment to operate in')
+    parser.add_argument('-e', '--environment', dest='env', default=current_environment(),
```

Comment:
Not really, because it's meant to be used by juju1 or juju2. I guess technically we could pick -e or -m at runtime depending on the output of `juju version`, but I'm not sure it's worth it. Sticking with -e follows a precedent set by juju-deployer and bundletester.